sample_workloads/lit-gpt-demo/slurm/launch_litgpt.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+#
+#SBATCH --partition=a3
+#SBATCH --exclusive
+#SBATCH --gpus-per-node 8
+#SBATCH --nodes 4
+
+export UDS_PATH="/run/tcpx-${SLURM_JOB_ID}"
+GPU_NIC_TOPOLOGY=/opt/tcpdirect_benchmark/gpu_rxq_configuration.textproto
+GPU_NIC_TOPOLOGY_DIR=`dirname ${GPU_NIC_TOPOLOGY}`
+
+if [ ! -f "${GPU_NIC_TOPOLOGY}" ]; then
+        echo "GPU_NIC_TOPOLOGY file ${GPU_NIC_TOPOLOGY} must exist!"
+        exit 1
+fi
+
+# Install NCCL plugin
+srun --ntasks-per-node=1 \
```

Comment:
Does srun output enough logging for user to debug?

Reply:
[srun](https://slurm.schedmd.com/srun.html) is used to run commands in parallel - The process that is being run will determine the amount of logging. But `srun` does have additional logging regarding the failure of running a command.